src/components/organisms/deployments/sessions/table/table.tsx

```diff
@@ -296,6 +292,13 @@ export const SessionsTable = () => {
 		navigateInSessions(filterEntityId, "");
 	};
 
+	const refreshViewer = async (): Promise<void> => {
+		if (sessionId) {
+			navigate(`./${sessionId}`, { state: { refreshViewer: Date.now() } });
+		}
+		refreshData();
+	};
```

Comment:
When we refresh the view, it always jumps to the first session in the list:

https://github.com/user-attachments/assets/8c1cc4b9-8529-4efb-8fa3-3be7616c4457